.github/workflows/enos-run.yml

```
@@ -100,31 +101,50 @@ jobs:
           cd /tmp/test-deps/pass/password-store-1.7.4
           sudo make install
           pass init ${{ secrets.ENOS_GPG_UID }}
-      - name: Download Vault AMD64 binary for integration testing
+      - name: Download Vault AMD64 binary
         if: steps.dep-cache.outputs.cache-hit != 'true'
         run: |
           wget https://releases.hashicorp.com/vault/1.12.2/vault_1.12.2_linux_amd64.zip -O /tmp/test-deps/vault.zip
-      - name: Install Vault for integration testing
-        if: matrix.filter == 'e2e_static_with_vault builder:crt' || matrix.filter == 'e2e_database'
+      - name: Install Vault CLI
+        if: matrix.filter == 'e2e_static_with_vault builder:crt' || matrix.filter == 'e2e_database' || matrix.filter == 'e2e_ui builder:crt'
         run: |
           unzip /tmp/test-deps/vault.zip -d /usr/local/bin
-      - name: Download Linux AMD64 Boundary bundle
+      - name: Download Boundary Linux AMD64 bundle
         id: download
         uses: actions/download-artifact@v3
         with:
           name: ${{ inputs.artifact-name }}
           path: ./enos/support/downloads
       - name: Unzip and rename Boundary bundle
         run: |
-          unzip ${{steps.download.outputs.download-path}}/*.zip -d enos/support
+          unzip ${{steps.download.outputs.download-path}}/*.zip -d enos/support/boundary
           mv ${{steps.download.outputs.download-path}}/*.zip enos/support/boundary.zip
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        if: matrix.filter == 'e2e_ui builder:crt'
+        with:
+          node-version: '16.x'
+      - name: Checkout boundary-ui
+        uses: actions/checkout@v3
+        if: matrix.filter == 'e2e_ui builder:crt'
+        with:
+          repository: hashicorp/boundary-ui
+          path: enos/support/boundary-ui
+      - name: Install boundary-ui dependencies
+        if: matrix.filter == 'e2e_ui builder:crt'
+        run: yarn --cwd enos/support/boundary-ui install
+      - name: Install playwright dependencies (i.e. browsers)
+        if: matrix.filter == 'e2e_ui builder:crt'
+        run: npx playwright install --with-deps
+        working-directory: enos/support/boundary-ui
       - name: Output Terraform version info
         # Use the same env vars from the following step
         env:
           ENOS_VAR_aws_region: us-east-1
           ENOS_VAR_aws_ssh_keypair_name: ${{ github.event.repository.name }}-ci-ssh-key
           ENOS_VAR_aws_ssh_private_key_path: ./support/private_key.pem
-          ENOS_VAR_local_boundary_dir: ./support/
+          ENOS_VAR_local_boundary_dir: ./support/boundary
+          ENOS_VAR_boundary_ui_dir: ./support/boundary-ui
```

Can we make this
```suggestion
          ENOS_VAR_local_boundary_ui_dir: ./support/boundary-ui
```
For consistency?